File: backend/RockClub.Shared/Dtos/ColaboradorResponseListDTO.cs

namespace RockClub.Shared.Dtos
{
    public class ColaboradorResponseListDTO
    {
        public Guid Id { get; set; }
        public string Nome { get; set; }
        public string Email { get; set; }
        public string Cargo { get; set; }
        public string Departamento { get; set; }
        public bool Status { get; set; }



        public ColaboradorResponseListDTO(Guid id, string nome, string email, string cargo, string departamento, bool status)
        {
            Id = id;
            Nome = nome;
            Email = email;
            Cargo = cargo;
            Departamento = departamento;
            Status = status;
        }
    }
}

File: frontend/src/app/models/ColaboradorResponseListDTO.ts

/*  no banco a resposta vem com as propriedades minusculas então aqui no angular coloque todas as response minusculo para evitar dor de cabeça */

export interface ColaboradorResponseListDTO {
  id?: string,
  nome: string,
  cargo: string,
  departamento: string,
  status: boolean
}
